doc/for-test-authors.rst

```diff
@@ -1385,6 +1385,10 @@ details of certain variables don't actually matter.
 See pages 419-423 of `xUnit Test Patterns`_ by Gerard Meszaros for a detailed
 discussion of creation methods.
 
+``testcase.generate_unique_text()`` is similar to ``getUniqueString``, except
+it generates text that contains unicode characters. As such the value will be a
+unicode string in Python2 and a str in Python3.
```

Comment:
This is confusing as written, perhaps:

The value will be a ``unicode`` object in Python 2, and a ``str`` object in Python 3.